libs/remix-ui/debugger-ui/src/lib/debugger-ui.tsx

```diff
@@ -8,6 +8,7 @@ import { DebuggerUIProps } from './idebugger-api' // eslint-disable-line
 import { Toaster } from '@remix-ui/toaster' // eslint-disable-line
 /* eslint-disable-next-line */
 import './debugger-ui.css'
+const helper = require('../../../../../apps/remix-ide/src/lib/helper')
```

Comment:
We can make this a standalone lib that can be imported and made use of by any app/lib.